### ISSUE TITLE
Use Robust Scaler and do not drop that much outliers

### ISSUE THREAD

**Issue by weisske** [Speedfan94/PDS_Project#55]
Robust Scaler can handle outliers much better than Standard Scaler. Therefore try not to drop 10% quantile but rather use Robust Scaler.

**Comment by MaxStroh** [comment]
Would have much influence we think, but we decided to mark this as optional to be able to prioritize other tasks

**Comment by weisske** [comment]
Robust scaler is worse than standard scaler on noise cleaned data.
NN_Regression Training loss - Error Metrics:
RMSE: 6.703324591964207 MAE: 4.379573332027776 R^2: 0.20181455956033834

performance on noisy data should be testet

**Comment by weisske** [comment]
Epoch 20/20
103608/103608 [==============================] - 19s 186us/sample - loss: 45.1719 - mae: 4.3037 - mse: 45.1719 - val_loss: 45.2533 - val_mae: 4.4006 - val_mse: 45.2533
